ui/apps/platform/src/Containers/NetworkGraph/common/FlowsTable.tsx

```diff
@@ -29,7 +29,7 @@ type FlowsTableProps = {
     flows: Flow[];
     numFlows: number;
     expandedRows: string[];
-    setExpandedRows: React.Dispatch<React.SetStateAction<string[]>>;
+    setExpandedRows?: React.Dispatch<React.SetStateAction<string[]>>;
```

Comment:
I'm curious why you marked this prop as optional, even though there is no other change that I can see in this PR where it's no longer passed.

Reply:
Oh, I think I tried to remove it but then decided not to do it in this PR. We technically aren't showing expanded rows anymore so I think there's some dead code in our Network Graph code right now